agent/xds/envoy_versioning_test.go

```diff
@@ -109,14 +109,7 @@ func TestDetermineSupportedProxyFeaturesFromString(t *testing.T) {
 	}
```

Comment:
I've been moving older versions up here into the "errTooOld" test list,so probably best to add all of the "1.15.x" series.